webdriver-spec.html

```diff
@@ -6443,14 +6443,14 @@
   <td>"<code>expiry</code>"
   <td>"<code>Max-Age</code>"
   <td>✓
-  <td>When the cookie expires,
-   specified in seconds since <a>Unix Epoch</a>.
-   Defaults to 20 years into the future
-   if omitted when <a>adding a cookie</a>.
+  <td>When the cookie expires, specified in seconds since
+   <a>Unix Epoch</a>.  Must not be set if omitted when
+   <a>adding a cookie</a>.
```

Comment:
The expiry is already marked as [optional in the table](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-table-for-cookie-conversion).